File: app/src/main/java/site/business/appslandz/retrobuilder/demo/Models/UsersList.java
package site.business.appslandz.retrobuilder.demo.Models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class UsersList {

    @SerializedName("data")
    public List<User> userList;

    public UsersList(List<User> userList) {
        this.userList = userList;
    }

    @Override
    public String toString() {
        return "UsersList{" +
                "userList=" + userList +
                '}';
    }
}

File: app/src/main/java/site/business/appslandz/retrobuilder/demo/Models/UsersList.kt
package site.business.appslandz.retrobuilder.demo.Models

import com.google.gson.annotations.SerializedName

class UsersList(@field:SerializedName("data") var userList: List<User>? = null) {

    override fun toString(): String {
        return "UsersList{" +
                "userList=" + userList +
                '}'
    }

}